src/components/Book.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeBook } from '../actions';
+
+/**
+ * Book
+ *
+ * Book component shows the information regarding
+ * a book
+ * @param {Object} props Data from parent
+ * @return {React.Component} Book
+ */
+const Book = props => {
```

Comment:
Let's keep our `book` component simple, screenshot from pathwright below;

![image](https://user-images.githubusercontent.com/31897434/82298659-56542080-99ac-11ea-9963-36e82ade1259.png)

Reply:
Solved!

We use book as prop:

https://github.com/mcastorena0316/Bookstore_CMS/blob/b833bc1e94a0d626112aa06353e5f49fefbbf10b/src/components/Book.js#L2-L14